source/AI.cpp

```diff
@@ -1773,16 +1775,17 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
-Point AI::StoppingPoint(const Ship &ship, bool &shouldReverse)
+// Instead of coming to a full stop, adjust to a target velocity vector
+Point AI::AdjustPoint(const Ship &ship, const Point &targetVelocity, bool &shouldReverse)
 {
 	const Point &position = ship.Position();
-	const Point &velocity = ship.Velocity();
+	const Point &velocity = ship.Velocity() - targetVelocity;
```

Comment:
Taking a reference to a temporary value is an error; I'm not sure why your compiler wouldn't flag this.